common/alignment/alignment-en.html

```diff
@@ -0,0 +1,98 @@
+---
+{
+	"title": "GCWeb Alignment example",
+	"language": "en",
+	"altLangPage": "alignment-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-06-19"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2>Horizontal Alignment</h2>
+<h3> Left aligned <code>text-left</code></h3>
+<p class="text-left">Use to align text to the left.</p>
```

Comment:
Considering that additional change is required, let's improve this working example to show it is simple not just falling back on the default text alignement behaviour. Something like:

```suggestion
<div class="text-right">
  <p class="text-left">Use to align text to the left.</p>
  <p>Text right aligned.</p>
</div>
```